compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -284,7 +284,7 @@ class Namer { typer: Typer =>
           case _ => (flags.isTermFlags, flags.toTermFlags, "value")
         }
         if (!ok)
-          ctx.error(i"modifier(s) `$flags' incompatible with $kind definition", tree.sourcePos)
+          ctx.error(i"modifier(s) `${flags.flagsString}' incompatible with $kind definition", tree.sourcePos)
```

Comment:
Should this not be:

```scala
`${flags.flagsString}`
```

That is, a backtick at the end rather than a single quote?